packages/react-ai/src/components/AIConversation/views/Controls/MessagesControl.tsx

```diff
@@ -16,7 +16,7 @@ const MESSAGE_BLOCK = 'ai-message';
 
 const MediaContent = withBaseElementProps(Image, {
   className: `${MESSAGE_BLOCK}__image`,
-  src: 'test.jpg',
+  alt: 'AI Image',
```

Comment:
Is this a placeholder? What is the context for this image (like how is it used)? Wondering if there is something more descriptive we should use.

Reply:
The image would be something the user attaches and sends. Since they upload using file input I'm not sure there's a way to really add descriptive alt text without adding a separate field for that. What do you think about `Image attachment` as the alt text?

Reply:
Yeah, that might make more sense. Whatever lines up with the file upload label maybe so the user can associate it to be the thing they just uploaded.